index.md

```diff
@@ -145,7 +145,7 @@ If you need to make quick fixes now, see <a href="https://www.w3.org/WAI/impl/im
 <p>There are evaluation  tools that  help with evaluation. However, no tool alone can determine if a site meets accessibility guidelines. Knowledgeable human evaluation is required to determine if a site is accessible.</p>
 
 {::nomarkdown}
-{% include box.html type="start" title="More Info on Evaluating" class="simple aside" %}
+{% include box.html type="start" h="3" title="More Info on Evaluating Accessibility" class="simple aside" %}
```

Comment:
Other box headings reflect the previous `<h2>` heading word by word as far as I could see, so I think it would be best practice to do this here, too.